test/controllers/api/v2/hostgroups_controller_test.rb

```diff
@@ -43,6 +43,14 @@ def valid_attrs_with_root(extra_attrs = {})
     assert show_response.has_key?('parameters')
   end
 
+  test "should show inherited parameters" do
+    get :show, params: { :id => hostgroups(:common).to_param }
```

Comment:
Maybe we could use `hostgroups(:inherited)` and validate even actual values:
```ruby

    assert_equal environments(:production).id, show_response['environment_id']
    assert_equal nil, show_response['inherited_environment_id']

    assert_equal nil, show_response['ptable_id']
    assert_equal templates(:autopart).id, show_response['inherited_ptable_id']
```

Reply:
Tip: there is `assert_nil` that can be used instead of `assert_equal nil, ...`

Reply:
@ezr-ondrej  @tbrisker done :)
thanks